cypress/utils/Util.ts

```diff
@@ -48,7 +48,8 @@ export function pageWasLoaded() {
   //If you remove this line the next waiting rutine (loading-table) was succesful even if the page (DOM) was empty
   //Tests was randomly failing that an element was reattached to DOM.
   cy.get("#nav-toggle", { timeout: 30000 }).should("be.visible");
-  cy.ouiaId("loading-table", "PF4/Card").should("not.exist");
+  //The mocked / dev should contain Instance one in the ready state, check that this instance is displayed correctly (no skeleton)
+  cy.get('div[class="pf-c-skeleton"]').should("not.exist");
```

Comment:
There are a couple of comments in the code referring to the `loading-table` selector. Could you fix them too?